### PR TITLE
Removes being able to emag drones.

### PR DESCRIPTION
**Pull request by awesine**
🆑 Awesine
add: Drones can no longer be emagged.
/🆑

Instead of nerfing them into being absolutely and utterly useless (#8499), just remove them.

Should be able to merge, not sure.

### CODE REVIEW

**Comment by taukausanake** [comment]
This was kind of done in #8440. I think I liked it more because of the popping. I don't think this will get merged that easily.

Also, using the web editor? Good luck getting _anything_ merged that way, maintainers hate it being used

**Comment by Tayyyyyyy** [comment]
We tried this already. Staff said they wanted it nerfed instead. I'll adjust the #8499 to staff feedback. Not ALL of those nerfs are going through.

**Comment by tigercat2000** [comment]
This doesn't even compile.

**Comment by Citinited** [comment]
Yeah, I'm not sure how this is going to be merged when a PR that did the same thing got closed less than a month ago. I do however agree that emagging drones should either be removed entirely or be something you buy as a standalone item for some amount of TC.

File: code/modules/mob/living/silicon/robot/drone/drone.dm [hunk 1]
```diff
@@ -126,7 +126,8 @@
 
 		if(stat == 2)
 
-			if(!config.allow_drone_spawn || emagged || health < -35) //It's dead, Dave.
+			if(!config.allow_drone_spawn || 
+			ged || health < -35) //It's dead, Dave.
```
Comment (Fox-McCloud) [inline code/modules/mob/living/silicon/robot/drone/drone.dm]: ....uh, what are you doing?

File: code/modules/mob/living/silicon/robot/drone/drone.dm [hunk 2]
```diff
@@ -236,22 +196,11 @@
 
 
 //CONSOLE PROCS
-/mob/living/silicon/robot/drone/proc/law_resync()
```
Comment (Tayyyyyyy) [inline code/modules/mob/living/silicon/robot/drone/drone.dm]: The removal of this proc could also be another factor in this not compiling...

**Comment by Ratvar** [comment]
... just redo  #8440

**Comment by tigercat2000** [comment]
Still doesn't compile, and this is unnecessary with #8499 being merged.

**Comment by awesine** [comment]
I just removed everything drone-emag related.

**Pull request closed without merging**